src/Api.ts

```diff
@@ -0,0 +1,16 @@
+const usersURL = 'https://jsonplaceholder.typicode.com/users';
+const todosURL = 'https://jsonplaceholder.typicode.com/todos';
```

Comment:
```suggestion
const API_URL = 'https://jsonplaceholder.typicode.com/';
```
можно использовать одну константу и дописывай где нужно